picoquic/frames.c

```diff
@@ -1113,7 +1116,7 @@ int picoquic_prepare_crypto_hs_frame(picoquic_cnx_t* cnx, int epoch,
     int ret = 0;
     picoquic_stream_head* stream = &cnx->tls_stream[epoch];
 
-    if ((stream->send_queue == NULL || stream->send_queue->length <= stream->send_queue->offset) && ((stream->stream_flags & picoquic_stream_flag_fin_notified) == 0 || (stream->stream_flags & picoquic_stream_flag_fin_sent) != 0)) {
+    if (stream->send_queue == NULL || stream->send_queue->length <= stream->send_queue->offset) {
```

Comment:
Just want to double-check that the omission of `&& (!stream->fin_requested || stream->fin_sent)` was deliberate.

Reply:
Yes it is. You cannot set the FIN bit on a crypto stream.